### ISSUE TITLE
Missing module

### ISSUE THREAD

**Issue by TechnoHT** [whatawurst/android_vendor_sony_poplar_kddi#5]
`FAILED: out/soong/build.ninja
out/soong/.bootstrap/bin/soong_build -t -l out/.module_paths/Android.bp.list -b out/soong -n out -d out/soong/build.ninja.d -globFile out/soong/.bootstrap/build-globs.ninja -o out/soong/build.ninja Android.bp
error: vendor/sony/poplar_kddi/Android.bp:88:1: module "com.fingerprints.extension" variant "android_common": module source path "vendor/sony/poplar_kddi/proprietary/framework/com.fingerprints.extension.jar" does not exist
error: vendor/sony/poplar_kddi/Android.bp:94:1: module "qcrilhook" variant "android_common": module source path "vendor/sony/poplar_kddi/proprietary/product/framework/qcrilhook.jar" does not exist
17:20:32 soong bootstrap failed with: exit status 1`

Is it normal to comment out these lines in Android.bp

**Comment by derfelot** [comment]
have you extracted the blobs before starting the build?

**Comment by TechnoHT** [comment]
> have you extracted the blobs before starting the build?

Can you give me a bit more detail
I usually leave everything at default when do `repo sync` and process to build like normal
this is the 1st time I build for the XZ1 KDDI and this error show up

**Comment by TechnoHT** [comment]
Successfully dumped both from a system dump of G8341 🤔 (compatibility still unknown)

**Comment by fadhsatr** [comment]
![image](https://user-images.githubusercontent.com/26964786/156971256-b5ab04bf-16f0-4ad3-9528-d4cee823a065.png)
I facing it too, how to fix it?

**Comment by TechnoHT** [comment]
> ![image](https://user-images.githubusercontent.com/26964786/156971256-b5ab04bf-16f0-4ad3-9528-d4cee823a065.png) I facing it too, how to fix it?

@fadhsatr run the extract script to get necessary blobs

**Comment by fadhsatr** [comment]
> @fadhsatr run the extract script to get necessary blobs

and now
![image](https://user-images.githubusercontent.com/26964786/157169404-b6154563-cc90-4a25-8ca3-dee1eb0ddabb.png)

**Comment by TechnoHT** [comment]
@fadhsatr I doubt you have to manually copy them over as if the script wasn't be able to pull them

**Comment by derfelot** [comment]
looks to me like you may have used the lineage-19.0 branch for device when extracting (I removed cacaoserver there).

**Comment by fadhsatr** [comment]
> looks to me like you may have used the lineage-19.0 branch for device when extracting (I removed cacaoserver there).

I already successfully compiled a rom, my fault before not using stock rom for extract-files.sh. another issue appear for me, when compile rom that not based on lineage the camera isnt working. any files that I should modify from lineage branch? thanks.

**Comment by TechnoHT** [comment]
> > looks to me like you may have used the lineage-19.0 branch for device when extracting (I removed cacaoserver there).
> 
> I already successfully compiled a rom, my fault before not using stock rom for extract-files.sh. another issue appear for me, when compile rom that not based on lineage the camera isnt working. any files that I should modify from lineage branch? thanks.

@fadhsatr you should open an issue since this will go nowhere I doubt
